packages/odf/components/s3-browser/create-or-edit-lifecycle-rules/create-lifecycle-rules.scss

```
@@ -11,3 +15,9 @@
         font-weight: var(--pf-t--global--font--weight--body--default);
     }
 }
+
+.s3-lifecycle-action-description {
+    white-space: normal;
+    word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)